### PR TITLE
2041 bug fix file with no participant

### PR DESCRIPTION
**Pull request by evans-g-crsj**
make sure that in the file repository screen file with no participant don't explode

### CODE REVIEW

File: src/components/EntityPage/EntityTitle.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ const EntityTitle = ({ icon, title, tags = [] }) => (
 EntityTitle.propTypes = {
   title: PropTypes.string,
   icon: PropTypes.string,
-  tags: PropTypes.arrayOf(PropTypes.string),
+  tags: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.string), PropTypes.array]),
```
Comment (jberube) [inline src/components/EntityPage/EntityTitle.js]: That's equivalent to `tags: PropTypes.array`

File: src/components/EntityPage/File/utils.js [hunk 2]
```diff
@@ -19,4 +19,19 @@ export const formatDate = date => {
   return `${yearDiff} years ${dayDiff} days`;
 };
 
-export const formatToGB = size => `${(size / 1000000000).toFixed(2)} GB`;
+//https://ourcodeworld.com/articles/read/713/converting-bytes-to-human-readable-values-kb-mb-gb-tb-pb-eb-zb-yb-with-javascript
+export const formatBytesToHumanReadable = (bytes, decimals = 2) => {
```
Comment (jberube) [inline src/components/EntityPage/File/utils.js]: 👌

File: src/components/EntityPage/EntityTitle.js (hunk 1 above)
Comment (evans-g-crsj) [reply src/components/EntityPage/EntityTitle.js]: corrected

File: src/uikit/CircleIcon.js [hunk 3]
```diff
@@ -35,7 +35,7 @@ CircleIcon.propTypes = {
   fill: PropTypes.string,
   size: PropTypes.number,
   iconSize: PropTypes.number,
-  Icon: PropTypes.element,
+  Icon: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
```
[the remark this answers is not in the document]
Comment (evans-g-crsj) [reply src/uikit/CircleIcon.js]: corrected

**Pull request merged**